instagram-app/src/components/PostContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import CommentSection from './CommentSection';
+import PropTypes from 'prop-types';
+import './PostContainer.css';
+
+function PostContainer(props){
+    return(
+        <div className="post-container">
+            
+            {props.data.map(post=>(
+                <div key={post.username} className="post">
+                    <div className="thumbnail"><img className="thumbnail-pic" src={post.thumbnailUrl} alt={post.name}/><h5>{post.username}</h5></div>
+                        <img className="main-pic"src={post.imageUrl} alt={post.name}/>
+                        <div className="btm-sec">
+                        <div className="pic-icons"><i class="far fa-heart"></i><i class="far fa-comment"></i></div>
+                        <p><strong>{post.likes} likes</strong></p>
+                        <div className="comments">
+                        {post.comments.map((e,i)=> <div><p key={i}><strong>{e.username}:</strong> {e.text}</p></div>)}
+                        </div>
+                    </div>
+                    <div className="commentos">
+                        <CommentSection />
+                    </div>
```

Comment:
You might want to pass an input prop into the `<CommentSection />` component - you will need a way to store inputs for new comments in your application state later on in order to be able to submit new comments.